source/agora/crypto/Schnorr.d

```diff
@@ -101,7 +101,7 @@ nothrow @nogc @safe unittest
 
 public Point extractNonce (Signature sig) @safe @nogc nothrow pure
 {
-    return Sig.fromBlob(sig).R;
+    return sig.R;
 }
```

Comment:
Maybe just remove this function then ? It doesn't seem to bring any value.

Reply:
Removed